zalo_sdk/BaseClient.py

```diff
@@ -23,11 +23,12 @@ def access_token(self):
     def access_token(self, token):
         self._access_token = token
 
-    def send_request(self, method, url, body):
+    def send_request(self, method: str, url: str, body: dict, xtra_headers: str = {}):
```

Comment:
typo? `xtra_headers`

Reply:
`xtra` is a shortcut for `extra` :P

https://www.dictionary.com/e/slang/xtra/

Reply:
whatttttt 🤯